app/api.py

```diff
@@ -1,192 +1,203 @@
-"""Simple Flask API"""
-
-from flask import Flask, jsonify, request, session, render_template
-from data import users, events, logged_users
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_migrate import Migrate
 from models import User, Event
-import os
-
-
 
 app = Flask(__name__)
 
-app.secret_key = os.urandom(24)
+app.config['SECRET_KEY'] = 'mysecretkey'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://marlone911:bev@localhost:5432/brightevents'
```

Comment:
It might be better to call this information from the .env file. It is also dangerous to push your DB credentials git.